src/utils/pointer-move-event-input.js

```diff
@@ -0,0 +1,40 @@
+import {PointerEventInput} from 'hammerjs';
+
+const POINTER_MOVE = 'pointermove';
+
+// Copied from Hammer.js' pointerevent.js
+const IE10_POINTER_TYPE_ENUM = {
```

Comment:
Question for @shaojingli - Should we draw the line at IE11? I am not convinced that IE10 reliably supports WebGL anyway. I think we never published our supported platforms doc.